web/src/lib/components/admin-page/settings/ffmpeg/ffmpeg-settings.svelte

```diff
@@ -126,9 +126,9 @@
             desc="VP9 has high efficiency and web compatibility, but takes longer to transcode. HEVC performs similarly, but has lower web compatibility. H.264 is widely compatible and quick to transcode, but produces much larger files."
             bind:value={ffmpegConfig.targetVideoCodec}
             options={[
-              { value: 'h264', text: 'h264' },
-              { value: 'hevc', text: 'hevc' },
-              { value: 'vp9', text: 'vp9' },
+              { value: VideoCodec.H264, text: 'h264' },
+              { value: VideoCodec.Hevc, text: 'hevc' },
```

Comment:
Is there a way to change the casing for this? I don't like how it's capitalized instead of uppercase.

Reply:
Not sure

Reply:
If I add `enumPropertyNaming=original` to the `--additional-properties` in `server/bin/generate-open-api.sh` and change enum values to uppercase:
```typescript
export enum VideoCodec {
  H264 = 'H264',
  HEVC = 'HEVC',
  VP9 = 'VP9',
}
```
I get the desired naming:
```typescript
/**
 * 
 * @export
 * @enum {string}
 */
export const VideoCodec = {
    H264: 'H264',
    HEVC: 'HEVC',
    VP9: 'VP9'
} as const;
```

But that's a global option. Not sure if we need this behavior for all enums. And also it seems we'll need a migration to convert user's ffmpeg options

Reply:
If I set `enumPropertyNaming=UPPERCASE`, I get the uppercase property names without changing the values. The only other enums affected are these:

```
export const AssetBulkUploadCheckResultActionEnum = {
  ACCEPT: 'accept',
  REJECT: 'reject',
} as const;

export type AssetBulkUploadCheckResultActionEnum =
  (typeof AssetBulkUploadCheckResultActionEnum)[keyof typeof AssetBulkUploadCheckResultActionEnum];
export const AssetBulkUploadCheckResultReasonEnum = {
  DUPLICATE: 'duplicate',
  UNSUPPORTED_FORMAT: 'unsupported-format',
} as const;
```

But that's probably a good thing since the TS enums for these also use uppercase anyway:

```
export enum AssetUploadAction {
  ACCEPT = 'accept',
  REJECT = 'reject',
}

export enum AssetRejectReason {
  DUPLICATE = 'duplicate',
  UNSUPPORTED_FORMAT = 'unsupported-format',
}
```

Reply:
Probably better to leave the casing alone for now and do this in a separate PR.